indra/llkdu/llimagej2ckdu.cpp

```diff
@@ -578,16 +579,18 @@ bool LLImageJ2CKDU::decodeImpl(LLImageJ2C &base, LLImageRaw &raw_image, F32 deco
 															mCodeStreamp.get()));
 				}
 				// Do the actual processing
-				F32 remaining_time = decode_time - decode_timer.getElapsedTimeF32();
+				F32 remaining_time = limit_time ? decode_time - decode_timer.getElapsedTimeF32() : 0;
```

Comment:
Use `0.0f` here to fix mac build error.

Reply:
Well, this was not enough :(
Probably casting to F32 or some refactoring will help.